app/file.txt

```diff
@@ -0,0 +1,4 @@
+123
+23
+33
+12
```

Comment:
You didn't have to add your test file to the pull request. :)
I'd actually even delete it.